pocket_snowflake/views/pocket_hits_r4_w_avg_click_counts.view.lkml

```diff
@@ -0,0 +1,38 @@
+view: pocket_hits_r4_w_avg_click_counts {
+  sql_table_name: "DBT_SHARED_WITH_MOZILLA"."POCKET_HITS_R4W_AVG_CLICK_COUNTS"
+    ;;
+
+  dimension: click_count {
+    type: number
+    sql: ${TABLE}."CLICK_COUNT" ;;
+  }
+
+  dimension: click_count_r4_w {
+    type: number
+    sql: ${TABLE}."CLICK_COUNT_R4W" ;;
+  }
+
+  dimension_group: happened_week {
+    type: time
+    timeframes: [
+      raw,
```

Comment:
@Marlene-M-Hirose is the data at the daily level? Not sure if we need the daily granularity in the dimension group because when I did it at the daily level I found there was a lot of 0s that made the visualizations weird.

Reply:
the data is at the weekly level. I will take out the raw and date timeframes

Reply:
Will add date filters to all of the explores in the model